benchmarks/dynamo/common.py

```diff
@@ -979,7 +979,7 @@ def setup_amp(self):
 
     def init_optimizer(self, name, device, params):
         if device == "cuda" and self.args.training and name not in CI_SKIP_OPTIMIZER:
-            self.optimizer = torch.optim.SGD(params, lr=0.01)
```

Comment:
interesting, SGD and Adam have different treatment of defaults wrt learning rate?

Reply:
Yeah, SGD is required, Adam has a default, not sure why lol